YAFC/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -880,6 +880,10 @@ private enum ProductDropdownType {
             }
         }
 
+        private static bool CheckIfOverproducing(ProductionLink link) {
+            return link.algorithm == LinkAlgorithm.AllowOverProduction && (link.flags & ProductionLink.Flags.LinkNotMatched) != 0;
```

Comment:
`LinkNotMatched` could also get triggered for/with `AllowOverConsumption`, so by just checking the flags it is not possible to determine over production. That is why I included the `element.linkFlow > element.amount` check for the coloring.

So the function name does not cover its implementation? Renaming to 'CheckPossibleOverproducing` might covert its implementation better?

Reply:
After a quick test: over consuming seems to cause the solver giving issues. So maybe this flag is never triggered? :thinking:

Reply:
Thank you for the suggestion! I've applied it in https://github.com/have-fun-was-taken/yafc-ce/pull/94/commits/e613047b3b75378854dd3abdbf1cdebbe6174c86 and https://github.com/have-fun-was-taken/yafc-ce/pull/94/commits/c89b7047f0a2ce9f585f5ba89c6d21459bc8e3c9.

Reply:
> seems to cause the solver giving issues

Could you please elaborate? Did you mean that the magenta color is not triggered for desired products when it should?

Reply:
The solver cannot solve overconsumption (there is no 'allow over consumption' option on a link). So the solver will abort with the error error box on top of the sheet. (same confusion as #96  is about, I think)